dashboard/test/ui/features/javalab/code_review.feature

```diff
@@ -3,18 +3,33 @@
 Feature: Code review
 
 Background:
-  Given I create a teacher named "Dumbledore"
-  And I create a new section
+  # Create a section
+  Given I create a levelbuilder named "Dumbledore"
+  And I sign in as "Dumbledore" and go home
+  And I create a new section named "CSA Section" assigned to "CSA Pilot"
+  And I save the section url
+  And I save the section id from row 0 of the section table
   Given I create a student named "Hermione"
   And I join the section
+  Given I create a student named "Harry"
```

Comment:
to slightly speed these up, would it be worth splitting this into 2 feature files: one for the tests that need both Hermione and Harry and one for the tests that only need Hermione? That way we won't waste time creating Harry for tests that don't need it.

Reply:
mmm most of the slowness is a result of the section creation flow (done via UI instead of via direct HTTP request) and navigation to teacher dashboard. My instinct is that the future confusion/duplication (why are these two different files?) isn't worth the speed gains we'd get.

Reply:
I saw a fairly significant speed-up when I moved the second create out of the setup steps (it's also through UI). I think if we name the files clearly it won't be too confusing. But if it's less than 30 seconds of speed then we can probably leave it.

Reply:
Yup you're totally right, shaved about a minute off! Split into two files, take a look and let me know what you think.

I could refactor some of the setup to be shared, but I'm tired of looking at these tests so I'm inclined to go with the write everything twice philosophy here.